src/shared/dynamodb.py

```diff
@@ -17,7 +17,7 @@
 logger = get_logger()
 
 
-def random_label(length):
+def random_label(length: int) -> str:
```

Comment:
nice improvement, courtesy of mypy.

Reply:
yes, so thankful for mypy and it's courteous behavior...